src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectTags.class.php

```diff
@@ -406,7 +409,7 @@ public function HasContent()
         return $bHasContent;
     }
 
-    protected function GetCleanedTagString()
+    protected function GetCleanedTagString(): string
```

Comment:
This is a BC break we should avoid.

Reply:
It is? But it always return string. removed it and added docblock instead.

Reply:
The BC breaks is that descendants of this class that override this method would be incompatible without also adding the return type declaration.